chirp/drivers/tdh8.py

```diff
@@ -1018,6 +1021,8 @@ def set_memory(self, mem):
             _mem.txfreq = (mem.freq + mem.offset) / 10
         elif mem.duplex == "-":
             _mem.txfreq = (mem.freq - mem.offset) / 10
+        elif mem.duplex == 'off':
+            _mem.txfreq = 166666665
```

Comment:
This value is larger than can be stored in the field, so it gets truncated. It also looks like an overflow value. What does the radio really want here? 0? 0xFFFF?